backends/gstreamer/audio_decoder.rs

```diff
@@ -145,7 +145,7 @@ impl AudioDecoder for GStreamerAudioDecoder {
 
             let insert_deinterleave = || -> Result<(), AudioDecoderError> {
                 let convert = gst::ElementFactory::make("audioconvert")
-                    // .property("mix-matrix", &gst::Array::new(&[]))
+                    .property("mix-matrix", &gst::Array::default())
```

Comment:
Why set this at all though?

Reply:
I was asking myself the same thing, but it was there in the previous version, so I just upgraded the syntax.